contracts/external/policies/ERC20SpendingLimitPolicy.sol

```diff
@@ -149,4 +120,52 @@ contract ERC20SpendingLimitPolicy is IActionPolicy {
             return VALIDATION_SUCCESS;
         }
     }
+
+    /**
+     * @notice Supports the IERC165 interface.
+     * @param interfaceID The interface ID.
+     * @return True if the interface is supported, false otherwise.
+     */
+    function supportsInterface(bytes4 interfaceID) external pure override returns (bool) {
+        return (
+            interfaceID == type(IERC165).interfaceId || interfaceID == type(IPolicy).interfaceId
+                || interfaceID == type(IActionPolicy).interfaceId
+        );
+    }
+
+    function _isTokenTransfer(
+        address account,
+        bytes calldata callData
+    )
+        internal
+        pure
+        returns (bool isTransfer, uint256 amount)
+    {
+        bytes4 functionSelector = bytes4(callData[0:4]);
+
+        if (functionSelector == IERC20.approve.selector) {
```

Comment:
should we add incrementApproval too? are there any other common approval funcs?

Reply:
I can't find incrementApproval in ERC20. Could you please link me, where is it defined and what is it applied to?

Reply:
sorry its `increaseAllowance` - used to be in [OZ](https://docs.openzeppelin.com/contracts/2.x/api/token/erc20#ERC20-increaseAllowance-address-uint256-) but now removed; I imagine that some ERC20s use this but dont have stats

Reply:
addressed here https://github.com/erc7579/smartsessions/pull/122/commits/a4d92df2967bae68f44169cd9af3d041a14d1907